frontend/src/redux/slices/MenuSlice.js

```diff
@@ -0,0 +1,112 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import { mockCategories as mockData } from '../../data/catalog/contact';

```

Comment:
I think it will be better to rename` contact.js` to `categories.js`. Maybe without using `catalog ` folder.
And maybe also rename `mockData` to `categoriesData`